tfhe/src/core_crypto/fft_impl/fft64/crypto/bootstrap.rs

```diff
@@ -245,24 +245,35 @@ pub fn bootstrap_scratch<Scalar>(
     )
 }
 
+pub fn blind_rotate_assign_scratch<Scalar>(
+    glwe_size: GlweSize,
+    polynomial_size: PolynomialSize,
+    fft: FftView<'_>,
+) -> Result<StackReq, SizeOverflow> {
+    StackReq::try_all_of([
+        StackReq::try_new_aligned::<Scalar>(polynomial_size.0 * glwe_size.0, CACHELINE_ALIGN)?,
+        cmux_scratch::<Scalar>(glwe_size, polynomial_size, fft)?,
+    ])
+}
```

Comment:
we already have the same scratch function line 220, this scratch is likely a wopbs specific scratch

Reply:
It's used in `tfhe/src/core_crypto/algorithms/lwe_programmable_bootstrapping/fft64.rs`

Reply:
this is unanswered

Reply:
Line 272

Reply:
looks like it might be an error in the import in the fft64 file as it should most likely use the scratch from this file